src/logic/operate.js

```diff
@@ -1,4 +1,4 @@
-import Big from 'big.js';
+import Big from 'big';
```

Comment:
You should import from `big-js` right?

You added dependency called `big-js`, so you should import from it.